src/averageAges.js

```diff
@@ -14,12 +14,28 @@
  *
  * @return {number}
  */
+
+function calculateAverage(list) {
+  const sum = list.reduce((sumPrev, person) => {
+    const PERSON_AGE = person.died - person.born;
+
+    return sumPrev + PERSON_AGE;
+  }, 0);
+
+  return sum / list.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((person) => person.sex === 'm');
+
+  if (century) {
```

Comment:
you should avoid `if` in this task - use the ternary operator
hint: you can define the full expression to filter by _inside_  filter's callback

Reply:
apply these changes everywhere